Framework/Algorithms/src/GetDetOffsetsMultiPeaks.cpp

```diff
@@ -1177,15 +1178,15 @@ void GetDetOffsetsMultiPeaks::addInfoToReportWS(
   m_infoTableWS->cell<double>(wi, 7) = offsetresult.highestpeakdev;
 
   // Peak width delta(d)/d
-  m_resolutionWS->dataX(wi)[0] = static_cast<double>(wi);
+  m_resolutionWS->mutableX(wi)[0] = static_cast<double>(wi);
```

Comment:
@LamarMoore But the histogram is of length one, so there is no way to do this better. This way off accessing is only a problem if you have `M` histograms of length `N`, with large `M` and `N`, since then `M * N` accesses will be expensive. If either `M` or `N` is small (say < 10) this is absolutely a non-issue.